modules/cluster-tagging.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="cluster-tagging_{context}"]
+= Tagging an Existing Cluster
```

Comment:
Was this module mistakenly added to this PR? I don't see how this module relates to RHEL workers.

Reply:
This was added in response to Comment 35 on the BZ ticket (https://bugzilla.redhat.com/show_bug.cgi?id=1872580#c35). Searching the documentation, there was nothing about tagging a cluster as owned, so I pulled this section out of 3.11.